src/jaspResults.cpp

```diff
@@ -2,11 +2,17 @@
 #include <fstream>
 #include <cmath>
 
+#ifdef BUILDING_JASP
 #include <boost/nowide/fstream.hpp>
 #include <boost/nowide/cstdio.hpp>
 
 typedef boost::nowide::ofstream bofstream; //Use this to work around problems on Windows with utf8 conversion
 typedef boost::nowide::ifstream bifstream;
+#else
```

Comment:
@JorisGoosen, I wonder what you think of this, and the one below for `boost::remove`.

Reply:
Why would we want different behaviour when not `BUILDING_JASP`? 
The rest of the code still assumes everything internally is utf-8 so this would break those kind of things.